README.md

```diff
@@ -45,11 +45,18 @@ or, if you haven't added your ssh key to your github account:
 git clone --recursive  https://github.com/Tribler/tribler.git
 ```
 
+Second install the [dependencies](https://github.com/Tribler/tribler/blob/devel/doc/Tribler%20development%20on%20Linux.md)
```

Comment:
Could you make this link relative so it points to the right version of the document when in different branches?